gallery/gallery/lib/l10n/gallery_localizations.dart

```diff
@@ -1812,6 +1812,58 @@ class GalleryLocalizations {
         desc: r'Title for the simple dialog component demo.');
   }
 
+  String get demoSnackbarsAction {
+    return Intl.message(r'You pressed the snackbar action.',
+        locale: _localeName,
+        name: 'demoSnackbarsAction',
+        desc:
+            r'Text that appears when you press on a snackbars action. Make sure to not translate index.');
+  }
+
+  String get demoSnackbarsActionButtonLabel {
+    return Intl.message(r'ACTION',
+        locale: _localeName,
+        name: 'demoSnackbarsActionButtonLabel',
+        desc: r'Label for action button text on the snackbar.');
+  }
+
+  String get demoSnackbarsButtonLabel {
+    return Intl.message(r'SHOW A SNACKBAR',
+        locale: _localeName,
+        name: 'demoSnackbarsButtonLabel',
+        desc: r'Label for button to show a snackbar.');
+  }
+
+  String get demoSnackbarsDescription {
+    return Intl.message(
+        r'Snackbars inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt the user experience, and they don’t require user input to disappear.',
+        locale: _localeName,
+        name: 'demoSnackbarsDescription',
+        desc: r'Description for snackbars demo.');
+  }
+
+  String get demoSnackbarsSubtitle {
+    return Intl.message(r'Snackbars show messages at the bottom of the screen',
+        locale: _localeName,
+        name: 'demoSnackbarsSubtitle',
+        desc: r'Subtitle for snackbars demo.');
+  }
+
+  String get demoSnackbarsText {
+    return Intl.message(r'This is a snackbar.',
+        locale: _localeName,
+        name: 'demoSnackbarsText',
+        desc:
+            r'Label for button to show a snackbar. Make sure to not translate index.');
```

Comment:
nit: Does this description need to be changed? It looks like it describes a button rather than snackbar content.

Reply:
Good call! Updated now.